submariner/OVN-Interconnect.md

```diff
@@ -0,0 +1,126 @@
+# Submariner Enhancement for OVN Interconnect
+
+<!-- Add link to issue/epic if available -->
+
+## Summary
+
+OVN CNI is moving to a new network topology powered by OVN Interconnect Feature. This allows independent
+OVN deployments to be interconnected by OVN-managed GENEVE tunnels. Submariner does not work with these changes
```

Comment:
Re: OVN-managed GENEVE - I am assuming OVN/OpenShift installer (and not Submariner) would need to make sure this traffic is allowed on firewalls?

Reply:
Yes that is right, OVN/Openshift installer should take care of it.